CONTRIBUTORS.md

```diff
@@ -7,6 +7,7 @@ Add yourself as a contributor to this project as a Markdown link that links your
 ## Cohort Participants
 
 Gabrielle Lommel (https://github.com/granolagabrielle) :seedling:
+Warren Chan (https://github.com/warrenchan13) 💪
```

Comment:
Hey, @warrenchan13! Almost there! Let's use the same syntax in the example above to get the markdown formatting:

[name](link) emoji

Could you also update @granolagabrielle entry? Thanks!

Reply:
Updated mine and @granolagabrielle's entry